DigitalSolutionsStudio.Api/D-Studio. Test task C# code sample.cs

```diff
@@ -0,0 +1,121 @@
+﻿private void btnGetData_Click(object sender, EventArgs e)
```

Comment:
1. The method is too large and should be split.
2. Change method name to a more revealing one.